tests/test_bgp.py

```diff
@@ -18,8 +17,7 @@
 from ax_interface.constants import PduTypes
 from sonic_ax_impl.mibs.ietf import rfc4363
 from sonic_ax_impl.main import SonicMIB
-from sonic_ax_impl.lib.quaggaclient import parse_bgp_summary
-from mock_tables.socket import MockGetHostname
+from sonic_ax_impl.mibs.vendor.cisco.bgp4 import CiscoBgp4MIB 
```

Comment:
>CiscoBgp4MIB [](start = 49, length = 12)

Not used?

Reply:
quaggaclient and mock tables are not required anymore.